demo/index.js

```diff
@@ -124,6 +126,16 @@ $(function() {
 
     function register(data) {
 
+        var outboundProxy =  localStorage.getItem('webPhoneOutboundProxy');
+        if(outboundProxy){
+            if(data.sipInfo[0]){
+                data.sipInfo[0].outboundProxy = outboundProxy;
+            }
+            if(data.sipInfo){
+                data.sipInfo.outboundProxy = outboundProxy;
+            }
```

Comment:
Please figure out under which conditions it is an array and when it is an object.